src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -192,25 +194,32 @@ public String  MakeGUID() {
     public void BeginConnectPeer(String toPeerId, ConnectStatusCallback connectStatusCallback) {
         ConnectStatusCallback tmpCallback = connectStatusCallback;
 
+        // needed since mBtToRequestSocket can be disconnected from other thread
+        BtToRequestSocket tmp = mBtToRequestSocket;
```

Comment:
@DrJukka Can you please use a more descriptive name like localMBtToRequestSocket?